### PR TITLE
Set up Ember Data, and use our backend for searching through municipalities

### PR DESCRIPTION
**Pull request by juliavdkris**
Also added a new migration file for `BestuurseenheidClassificatieCode`s, so we can query just municipalities

### CODE REVIEW

**Review by D34DPlayer — CHANGES_REQUESTED** [review]
Answer my prayers

File: backend/dispatcher/dispatcher.ex [hunk 1]
```diff
@@ -14,6 +14,15 @@ defmodule Dispatcher do
   @json %{ accept: %{ json: true } }
 
 
+  # Handle OPTIONS preflight HTTP request
+  options "/*" do
+    conn
+    |> Plug.Conn.put_resp_header( "access-control-allow-headers", "content-type,accept" )
+    |> Plug.Conn.put_resp_header( "access-control-allow-methods", "*" )
```
[the remark this answers is not in the document]
Comment (juliavdkris) [reply backend/dispatcher/dispatcher.ex:21]: Ahh that might've actually been why CORS wasn't working without the identifier yeah. Either way, the identifier fixed everything (including cancer, war, and world hunger) so we're good now :)

File: frontend/app/models/tax-report.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import Model, { attr, hasOne } from '@ember-data/model';
+
+export default class TaxReportModel extends Model {
+	@attr('number') year;
+	@attr('number') amount;
+	@hasOne('mar-code') MARCode;
+	@hasOne('bestuursorgaan') municipality;
```
[the remark this answers is not in the document]
Comment (juliavdkris) [reply frontend/app/models/tax-report.js:7]: Good catch! I'll quickly fix that, thanks!

**Review by D34DPlayer — APPROVED** [review]
Looks good now, good job!

**Pull request merged**